src/components/AppointmentConfigModal.vue

```diff
@@ -201,6 +202,7 @@ export default {
 			enableFutureLimit: false,
 			rateLimitingReached: false,
 			showConfirmation: false,
+			store: useAppointmentConfigsStore(),
```

Comment:
I'd prefer to retain the name of the store to make it more readable, e.g. `appointmentConfigStore`. It is trivial in this component but it might not be so obvious in others.

You could use the `mapStore` helper to add stores to components.

```js
import { mapStores } from 'pinia'
import useAppointmentConfigsStore from '../store/appointmentConfigs.js'

export default {
  computed: {
    ...mapStores(useAppointmentConfigsStore)
  },

  methods: {
    async foo() {
        await this.apointmentConfigStore.bar()
    },
  },
}
```